### DISCUSSION TITLE
Learn By Building Projects, And Explain It To Others

### DISCUSSION THREAD

**Discussion by abderrahmaneMustapha** [dzcode-io/dzcode.io#380] [Ideas]
i have an idea similar to[ project learn](https://projectlearn.io/#categories) and [codementor](https://www.codementor.io/projects)  we can add it to dzcode

where each contributor can create a simple project + an article or a video showing how he did that

and this blog/video is gonna be like an inspiration/guide for other  algerian developers, to learn while building something.

**Comment by omdxp** [comment]
@abderrahmaneMustapha great idea, looking forward to apply it!!

**Comment by abderrahmaneMustapha** [comment]
@ZibanPirate  and @Omar-Belghaouti  do you have any ideas ????

**Reply by omdxp** [reply]
How about just using the articles section? And if he want to use a video he can use a YouTube link within?

**Reply by abderrahmaneMustapha** [reply]
@Omar-Belghaouti  great idea , for me i think that building a different page with a different user interface  + a  search  feature  and filters similar to  [dzcode contribute page](https://www.dzcode.io/Contribute)  will attract the website visitor, if you want us to use the articles page i think that we need to edit the ui , what do you think ???

**Reply by omdxp** [reply]
@abderrahmaneMustapha I think that building a different page with new ui and custom feature will be better, because the article pages should be simple as it is for now, but it can be also linked to that page like what we're doing in the fag page where we have some links to other pages within the website

**Comment by souhaib-benbouzid** [comment]
I love the idea. any suggestions on how would we implement it?

**Comment by CA1R7** [comment]
@abderrahmaneMustapha  Great idea, It'll help so much peoples also will make dzcode greatest.

**Comment by abderrahmaneMustapha** [comment]
perfect ,  @Omar-Belghaouti  @Souhaib-Benbouzid  @CA1R7  @ZibanPirate   i will share with you some more ideas after completing  the [Implement localization](https://github.com/dzcode-io/dzcode.io/issues/24) issue, and if you have some ideas about how can we implement/improve this  please share them with us
